PROJECT

```diff
@@ -34,6 +34,15 @@ resources:
   kind: LicenseMaster
   path: github.com/splunk/splunk-operator/api/v3
   version: v3
+- api:
+    crdVersion: v1
```

Comment:
Although this is the first version of the LicenseManager CR, in the past, we have tried to align the new CRs with versions of the existing CRs (MC CRD is an example).
cc: @kashok-splunk , @vivekr-splunk

Reply:
Agreed. I will double-check if we need to change something. Currently, it looks like this new CRD is set to v3.

Reply:
Confirmed we don't need additional changes for now as the CRD is on `V3`. I think that `v1` above is related to `apiextensions` version. Our CRD is referenced in line 45 as `version: v3`. 
cc: @kashok-splunk , @vivekr-splunk